chroma-agent/chroma-agent.spec

```diff
@@ -27,6 +27,7 @@ Requires: python-requests >= 2.6.0
 Requires: python-tablib yum-utils
 Requires: initscripts
 Requires: chroma-diagnostics >= %{version}
+Requires: chroma-common >= %{version}
```

Comment:
The diagnostics is pretty (albeit not completely) independent from both chroma-agent and chroma-management.  I think >= for it is still fine.